reconcile/test/test_utils_oc.py

```diff
@@ -267,6 +267,58 @@ def test_controller_true_allow_false_ref_not_found_raise(self, oc_get):
             oc.get_obj_root_owner('namespace', obj, allow_not_found=False)
 
 
+@patch.dict(os.environ, {"USE_NATIVE_CLIENT": "False"}, clear=True)
+class TestPodOwnedPVCNames(TestCase):
+    def test_no_volumes(self):
+        pods = [{'volumes': []}]
+        oc = OC('cluster', 'server', 'token', local=True)
+        owned_pvc_names = oc.get_pod_owned_pvc_names(pods)
+        self.assertEqual(len(owned_pvc_names), 0)
```

Comment:
Better:
```python
self.assertFalse(owned_pvc_names)
```
it'll show how `owned_pvc_names` is true-y, making debugging easier.

Reply:
not sure i agree with that (or maybe i'm not understanding something).
the test is called `test_no_volumes`, which to me makes a lot of sense that it means that the length of the returned list will be 0.